library/php

```diff
@@ -149,17 +199,17 @@ Architectures: amd64, arm32v5, arm32v7, i386
 GitCommit: 2d413e2dff930b1ac261a587cb8911df2c6bd355
 Directory: 7.0/jessie/zts
 
-Tags: 7.0.31-cli-alpine3.7, 7.0-cli-alpine3.7, 7.0.31-alpine3.7, 7.0-alpine3.7, 7.0.31-cli-alpine, 7.0-cli-alpine, 7.0.31-alpine, 7.0-alpine
+Tags: 7.0.31-cli-alpine3.7, 7.0-cli-alpine3.7, 7.0.31-alpine3.7, 7.0-alpine3.7
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
 GitCommit: 2d413e2dff930b1ac261a587cb8911df2c6bd355
 Directory: 7.0/alpine3.7/cli
 
-Tags: 7.0.31-fpm-alpine3.7, 7.0-fpm-alpine3.7, 7.0.31-fpm-alpine, 7.0-fpm-alpine
+Tags: 7.0.31-fpm-alpine3.7, 7.0-fpm-alpine3.7
```

Comment:
Is it intentional that the `7.0-fpm-alpine` tags have been removed (because there is no Alpine 3.8 for 7.0).